src/controller/auth.js

```diff
@@ -10,7 +10,12 @@ const unauthError = require("../responses/unauthenticated.json");
 module.exports = router
 	.post("/register", (req, res) => {
 		registerAccount(req.body.username, req.body.password).then(response => {
-			res.json(response);
+			if (response.ok = false) {
```

Comment:
`if(!response.ok)` would be cleaner, apart from you not checking for `false`, but assigning `false` to `response.ok`.